internal/rbd/replicationcontrollerserver.go

```diff
@@ -450,6 +455,19 @@ func (rs *ReplicationServer) ResyncVolume(ctx context.Context,
 		}
 	}
 
+	// when the images are demoted on both clusters and user requests for the
+	// resync of the image, the image mirror state will be unknown state in
+	// both clusters.
+	if state == upAndUnknown {
+		ready = true
```

Comment:
We don't need to set it true here.
I am thinking of a scenario when
1. it was false because of the above code
2. comes here and becomes true
3. never goes to 466
this will result in ready becoming "true" even though it should be "false"

Ideally if ready is "false" because of 453, we should not even check this.

Reply:
this is set to make it true when the image state is unknown in both cases. it will be set to false even if one of the remote status is not up and unknown.  if the state is set at 453 we are not going to enter this loop.

Reply:
Yeah, it will not come here in that case. sorry for the confusion.